presets/all-packages/src/index.ts

```diff
@@ -0,0 +1 @@
+export default {};
```

Comment:
We don't want to export everything at once. Consumers will need to import whatever they need independently.